llvm/test/Assembler/repo-prunning.ll

```diff
@@ -15,24 +15,19 @@
 
 target triple = "x86_64-pc-linux-gnu-elf"
 
-;CHECK: @c = available_externally global i32 8, align 4
 $c = comdat any
```

Comment:
We no longer check for `available_externally`. That's presumably because these variables and functions are now removed completely. Could we check for that instead?

Reply:
Yes, added some checks.